commit 59448fccf2982e6859ea42b66ec4df8f57c73be0
Author: John Levon <john.levon@joyent.com>
Date:   2019-08-22T16:37:24+00:00 (7 weeks ago)
    
    MANTA-4489 ops is under-sized in a lab deployment
    Reviewed by: Trent Mick <trentm@gmail.com>
    Reviewed by: Isaac Davis <isaac.davis@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/config/services/ops/service.json.lab b/config/services/ops/service.json.lab
new file mode 100644
index 0000000..2a27ab7
--- /dev/null
+++ b/config/services/ops/service.json.lab
@@ -0,0 +1,5 @@
+{
+	"params": {
+		"ram": 4096
+	}
+}
